cmd/meroxa/root/apps/deploy.go

```diff
@@ -203,14 +205,19 @@ func (d *Deploy) createApplication(ctx context.Context, pipelineUUID, gitSha str
 				return err
 			}
 			if app.Pipeline.UUID.String != pipelineUUID {
-				return fmt.Errorf("unable to finish creating the %s Application because its entities are in an"+
-					" unrecoverable state; try deleting and re-deploying", appName)
+				d.logger.StopSpinner(fmt.Sprintf("\t  𐄂 unable to finish creating the %s Application because its entities are in an"+
+					" unrecoverable state; try deleting and re-deploying", appName))
+
+				// TODO: Rollback here?
```

Comment:
oh, so rollback is a feature of the CLI, not of the language specific libraries? or both?

Reply:
@janelletavares it should be library specific I think